pixels/canvas.py

```diff
@@ -40,28 +40,28 @@ async def _populate_cache(self, conn: Connection) -> None:
         """Populate the cache and discard old values."""
         start_time = time()
 
-        cache = bytearray(constants.width * constants.height * 3)
+        cache = bytearray(Sizes.WIDTH * Sizes.HEIGHT * 3)
 
         sql = (
             "SELECT x, y, rgb "
             "FROM current_pixel "
             "WHERE x < $1 "
             "AND y < $2"
         )
```

Comment:
This is purely for readability, I like having SQL queries like this as each logical step is on a new line.

I may change other SQL queries in the project to this format for consistency.

Reply:
Did using multiline strings cause issues?

Reply:
Neither causes issues, but we had a few different styles for doing multi-line queries, so just made them all consistent.

This one specifically could actually be flattened, since it's not a complex queries like in other places.